Introducing GDPR compliance checks and policy (#10064)
JabRef/jabref: PRIVACY.md

The Browser Extension does not share any data except with the local instance of the JabRef software application, which stores the citation data as a new entry in its library.

## Links to other Websites
- Our Service may contain links to other websites that are not operated by Us. If You click on a third party link, You will be directed to that third party's site. We strongly advise You to review the Privacy Policy of every site You visit.
- We have no control over and assume no responsibility for the content, privacy policies or practices of any third party sites or services.

## Changes to this Privacy Policy
This privacy policy may be changed eventually. 
We encourage you to check this Privacy Policy periodically for any changes.
Any material change will be mentioned in the changelog and in our blog.

This privacy policy is in effect as of the day mentioned as "last updated" above and will remain in effect except with respect to any changes in its provisions in the future, which will be in effect immediately after being posted on this page.

## Contact
For any questions or concerns regarding the privacy policy, please send us an email to <vorstand@jabref.org> or write to

> JabRef e.V.<br>

Reviewer: <br> in markdown?

Author: https://www.markdownguide.org/basic-syntax#line-break-best-practices

# Privacy Policy
Last updated: 2023-08-24

Your privacy is a fundamental right JabRef e.V. respects and supports.

> By using JabRef and its related online services, you choose to share some of this information.

Reviewer 2: What refers 'this' to, reorder a bit here?

> In this Privacy Policy We explain how We collect, use, and share information about you, along with the choices you have.

Reviewer 2: We, Us and Privacy Policy should follow standard English capitalization rules (or at least be consistently written)

Author: I believe "Privacy Policy" is a fixed term and a name, so I uppercased it in the document, but I'm not sure.

Please remember that no method of transmission over the Internet, or method of electronic storage is absolute secure.
While We strive to use commercially acceptable means to protect your Personal Data, We cannot guarantee its absolute security.
Also, whenever you communicate through the internet, your IP-Address will be transmitted and retained by other parties for technical reasons.

> The term 'Personal information' in this policy means any information that either directly identifies you or can be somehow linked to you. 'JabRef' is the software application, JabRef e.V. provides.

Reviewer 2: Move the last sentence to the next subsection.

> The term 'Personal information' in this policy means any information that either directly identifies you or can be somehow linked to you. 'JabRef' is the software application, JabRef e.V. provides.

Reviewer 2: ```suggestion
The term 'Personal information' in this policy means any information that either directly identifies you or can be somehow linked to you. 'JabRef' refers to the desktop application that is provided by JabRef e.V.
```

> ## JabRef

Reviewer 2: ```suggestion
## Desktop Application 
```

### Collecting information
JabRef does not collect any personal information directly linked to you.
However, on certain occasions the software application will send some information to the online services of JabRef e.V.:
- On application start, JabRef will check for the latest version online (by default *enabled*).
- Information about a journal you are citing when looking for more information about this journal, using our journal database (by default *enabled*).
- A pdf document you automatically want to extract citation information from, using our GROBID service (by default *disabled*).
- Anonymized statistical data on the use of the graphical user interface for internal analysis purposes (by default *disabled*).

### Storing information

> JabRef does not store any personal information, except:

Reviewer 2: ```suggestion
JabRef only stores the following personal information locally on your computer:
```

- Your proxy username and password, if you decide to store them (by default *disabled*).
- Any personal API key you use to access third party online services (by default *disabled*). 

### Sharing information
Certain operations you perform in JabRef may trigger requests to public third-party services such as Zotero, Crossref or the Library of Congress for metadata retrieval.

> These third parties may log besides your IP address and the search terms (e.g., DOI, ISBN or the current URL) according to their privacy policies.

Reviewer: ```suggestion
These third parties may log additional information besides your IP address and the search terms (e.g., DOI, ISBN or the current URL) according to their privacy policies.
```

Reviewer 2: I would emphasize that we don't send any personal information to these services, except for what is strictly necessary to perform the task. In particular, only the ip Adress and the provided search terms are thansfered.

| [Semantic Scholar](https://www.semanticscholar.org/), powered by [Allen Institute for AI](https://allenai.org/) | <https://allenai.org/privacy-policy> |
| [Springer Nature](https://dev.springernature.com/)                                                              | <https://dev.springernature.com/privacypolicy> |
| [The SAO/NASA Astrophysics Data System](https://ui.adsabs.harvard.edu/)                                         | <https://ui.adsabs.harvard.edu/help/privacy/> |
| [Unpaywall](https://unpaywall.org/)                                                                             | <https://unpaywall.org/legal/privacy> |
| [zbMATH Open](https://www.zbmath.org)                                                                           | <https://zbmath.org/privacy-policy/> |

&ast; *Note: The Mr. DLib service is used for the related articles tab in the entry editor and collects also your language, your browser and operating system (by default *disabled*).* 

## JabRef Browser Extension

No personal data (like name, email address, billing address or credit card) is collected.

> By using the extension you will remain completely anonymous.

Reviewer: I would remove this sentence

### Collecting information

When you access a website with your web browser, the Browser Extension is designed to save time by allowing you to quickly import bibliographic information into JabRef.

> Therefor the extension collects and analyzes some information concerning your browser activity.

Reviewer: The extension does not really collect or store information. 
Suggestion: 
When actively used, the extension has access to the current website and its content. The information is only for the purpose of processing citations.

## Links to other Websites
Our Service may contain links to other websites that are not operated by Us. If you click on a third party link, you will be directed to that third party's site. We strongly advise you to review the Privacy Policy of every site you visit.

We have no control over and assume no responsibility for the content, privacy policies or practices of any third party sites or services.

## Changes to this Privacy Policy
This privacy policy may be changed eventually. 
We encourage you to check this Privacy Policy periodically for any changes.
Any material change will be mentioned in the changelog and in our blog.

This privacy policy is in effect as of the day mentioned as "last updated" above and will remain in effect except with respect to any changes in its provisions in the future, which will be in effect immediately after being posted on this page.

## Contact
For any questions or concerns regarding the privacy policy, please send us an email to <vorstand@jabref.org> or write to

JabRef e.V.<br>
Josef-Lanner-Str. 9<br>
71069 Sindelfingen<br>
Germany

> If you get in touch with us, We may aks you to provide Us with certain personal information (e.g. name and email address) that can be used to contact or identify you.

Reviewer 2: That last sentence feels very much out of place here. Just delete it?

Author: Should go somewhere, because that is in fact also an occasion, we collect and store data.

Reviewer 2: Then it should be a complete new section...
